### PR TITLE
serf: fix for Linux

### PR DESCRIPTION
**Pull request by maxim-belkin**
Had to do this change to make `serf` compile.

### CODE REVIEW

**Comment by sjackman** [comment]
I think this `serf` formula is different than the `serf` mentioned in `subversion`. This one is https://serfdom.io/
The one in `subversion` is http://serf.apache.org/

File: Library/Formula/serf.rb [hunk 1]
```diff
@@ -106,7 +106,11 @@ def install
     ENV["GOPATH"] = gopath
     arch = MacOS.prefer_64_bit? ? "amd64" : "386"
     ENV["XC_ARCH"] = arch
-    ENV["XC_OS"] = "darwin"
+    if OS.mac?
+      ENV["XC_OS"] = "darwin"
+    else
+      ENV["XC_OS"] = "linux"
+    end
```
Comment (sjackman) [inline Library/Formula/serf.rb]: Please change this to
```ruby
ENV["XC_OS"] = OS::NAME
```

**Comment by sjackman** [comment]
Thanks for the fix, Maxim. I'll be happy to merge it after the suggested change.

**Comment by maxim-belkin** [comment]
Sorry for the delay. I've updated the commit.

**Comment by sjackman** [comment]
Unrelated error when installing `curl`
https://travis-ci.org/Linuxbrew/linuxbrew/builds/111801055#L1229
```
/usr/bin/perl ./zsh.pl ../src/curl > _curl
/tmp/curl20160225-30127-6bwhx2/curl-7.47.1/src/.libs/lt-curl: error while loading shared libraries: liblber-2.4.so.2: cannot open shared object file: No such file or directory
```
`liblber` is related to LDAP. https://packages.debian.org/sid/amd64/libldap-2.4-2/filelist

**Comment by sjackman** [comment]
No worries. Merged. Thanks for your contribution, Maxim.

**Pull request closed without merging**